libs/components/storybook/src/lib/preview-wrapper/preview-wrapper.component.ts

```diff
@@ -45,40 +45,46 @@ export class PreviewWrapperComponent implements OnInit, OnDestroy {
     }
   }
   public get theme() {
-    return this._theme;
+    return this.#_theme;
   }
 
   public get themeSettings(): SkyThemeSettings {
-    return this._themeSettings;
+    return this.#_themeSettings;
   }
   public set themeSettings(value: SkyThemeSettings) {
-    this._themeSettings = value;
-    if (this.initialized) {
-      this.themeService.setTheme(this._themeSettings);
+    this.#_themeSettings = value;
+    if (this.#initialized) {
+      this.#themeService.setTheme(this.#_themeSettings);
     }
   }
 
-  private _themeSettings = new SkyThemeSettings(
+  #_themeSettings = new SkyThemeSettings(
     SkyTheme.presets.default,
     SkyThemeMode.presets.light
   );
-  private _theme?: 'default' | 'modern-light' | 'modern-dark';
-  private readonly _ngUnsubscribe = new Subscription();
-  private initialized = false;
+  #_theme: 'default' | 'modern-light' | 'modern-dark' = 'default';
```

Comment:
Can we make this string union an official type, or at least a type defined at the top of this file?

Reply:
Done. I went ahead and just did what we do for the theme selector in the apps and made a local type. I wonder if we should have a more global way to do this defined in `theme`, but I didn't want to deal with that at this point.